ddl/schema.go

```diff
@@ -81,7 +81,7 @@ func (d *ddl) onDropSchema(t *meta.Meta, job *model.Job) error {
 	}
 	if dbInfo == nil {
 		job.State = model.JobCancelled
-		return errors.Trace(infoschema.ErrDatabaseDropExists)
+		return infoschema.ErrDatabaseDropExists.Gen("Can't drop database; database doesn't exist")
```

Comment:
It's better to create a error like "ErrDatabaseDropExistsWithoutColName", or use errors.New(). `Gen` is not recommanded.